common/components/common/projects/EditProjectForm.jsx

```diff
@@ -221,7 +221,7 @@ class EditProjectForm extends React.PureComponent<Props,State> {
 
   _renderForm(): React$Node {
     return (
-      <div className="EditProjectForm-root">
+      <div className="EditProjectForm-root create-form white-bg container-fluid">
```

Comment:
We don't need this `.container-fluid`declaration because EditProjectController already encases this entire component in a `.container` class.